src/js/player.js

```diff
@@ -2381,6 +2382,26 @@ class Player extends Component {
   }
 
   /**
+   * Get or set the playsinline attribute.
+   *
+   * @param {boolean} [value]
+   *        - true means that we should play inline in iOS Safari
```

Comment:
I would change these two say `set the playsinline attribute` or `unset it`. And mention above that this probably only applies on ios right now.